src/VSCodeExtension/packages/extension/package.json.v.template

```diff
@@ -97,7 +97,7 @@
         ]
     },
     "scripts": {
-        "vscode:prepublish": "npm install && npm run compile",
+        "vscode:prepublish": "npm install ../generator-qsharp/ && npm install && npm run compile",
```

Comment:
That's one way to do it, should definitely work around most of the issues we ran into with lerna and similar approaches!